index.bs

```diff
@@ -89,6 +93,33 @@ spec: ECMAScript; urlPrefix: https://tc39.github.io/ecma262/#
     overflow: hidden;
     word-break: normal;
   }
+
+  .unstable::before {
+    content: "This section is not stable";
+    display: block;
+    font-weight: bold;
+    text-align: right;
+    color: red;
+  }
+
+  .unstable {
+    border: thin solid pink;
+    border-radius: .5em;
+    padding: .5em;
+    margin: .5em calc(-0.5em - 1px);
+    background-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='300' height='290'><text transform='rotate(-45)' text-anchor='middle' font-family='sans-serif' font-weight='bold' font-size='70' y='210' opacity='.1'>Unstable</text></svg>");
+    background-repeat: repeat;
+    background-color: #FFF4F4;
+  }
+
+  .unstable h3:first-of-type {
+    margin-top: 0.5rem;
+  }
+
+  .unstable.example:not(.no-marker)::before {
+    content: "Example " counter(example) " (Unstable)";
+    float: none;
+  }
```

Comment:
Do we need all this? What's the default style bikeshed provide? Or does it not provide anything?

Reply:
It doesn't seem to provide anything. I've followed the same approach that the main WebXR spec was taking when it still had some unstable parts of the API (all those CSS entries were originally present in this .bs file).